elisa/src/client/RunDetails/components/CurveFitPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from 'react';
+
+import { CurveFitData, PlotOptions } from "../models";
+import { CurveFitDataDisplay } from "./CurveFitDataDisplay";
+import { LabelHelpTip } from "@labkey/components";
+import { DEFAULT_X_AXIS_PROP, DEFAULT_Y_AXIS_PROP } from "../constants";
+
+interface Props {
+    runPropertiesRow: {[key: string]: any},
+    plotOptions: PlotOptions,
+    setPlotOption: (key: string, value: any, resetIdSelection: boolean) => void,
+    curveFitData: CurveFitData
+}
+
+export class CurveFitPanel extends PureComponent<Props> {
+
+    onShowCurveCheck = (evt) => {
+        this.props.setPlotOption('showCurve', evt.target.checked, false);
+    };
+
+    render() {
+        const { plotOptions, runPropertiesRow, curveFitData } = this.props;
+
+        return (
+            <div className="panel panel-default">
+                <div className="panel-heading">
+                    Curve Fit: {runPropertiesRow?.CurveFitMethod ?? 'Linear'}
+                </div>
+                <div className="panel-body">
+                    <div className={'plot-options-section'}>
+                        <div>
+                            <input
+                                type="checkbox"
+                                name='showCurve'
+                                checked={plotOptions.showCurve}
+                                onChange={this.onShowCurveCheck}
+                            />
+                            Show curve fit line
+                            <LabelHelpTip
+                                title={'Standard Curve Fit'}
+                                body={() => (
```

Comment:
Looks like LabelHelpTip is a PureComponent.  By redeclaring the function here with every render it's causing the body prop to always be different and unnecessary re-renders of LabelHelpTip

Reply:
done